.github/workflows/gotest.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
         with:
-          go-version: 1.19
+          go-version-file: 'go.mod'
```

Comment:
Might be good here to use `1.21` and create a separate issue and update it globally across all the devfile repos?

Reply:
@thepetk Since this is just a single module repo, I thought I'd use `go-version-file` instead to have it use the same Go runtime defined by the module, such as what is done in devfile/library: https://github.com/devfile/library/blob/6a00bfa4a09e44bf2647682b06f19182c585fe0a/.github/workflows/go.yml#L43

With `go.mod` updated to `1.21`, this version should also be used for the environment as well.

Reply:
Hmm.. when I commented I was mainly thinking about alizer

https://github.com/devfile/alizer/blob/main/.github/workflows/CI.yml#L17

Reply:
> in all repos that its needed.

Yes sorry my bad. I should have written it more explicitly. Totally agree that we should avoid adding it on registry-support and registry. We should only added to repos that are a single module (e.g having one root go.mod)

> I could change this to explicitly set the Go version and change this later in a separate issue for all repos (excluding library which has this already) or leave it to reduce the places we need to make this update to later. WDYT?

Maybe we could change this one and do a quick check for the others (we could open a small issue). I think is a good addition.